README.md

```diff
@@ -1,37 +1,58 @@
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Set up a database
-<br />
-<br />&nbsp;&nbsp;    ○3-7 "standard" requirements (will most likely complete)
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Admins should be able to add new book/movie manually
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     The website should be able to update its book/movie list via external API endpoints
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Allow (regular?) users to add a review & ratings (out of 5 starts) for a book/movie
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Users should be able to add a book or movie to a personal 'Watch later' list
-<br />
-<br />&nbsp;&nbsp;    ○2-3 stretch requirements (plan to complete 1!)
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■    Locate a theater showing chosen movies
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■    Users should be able to see information of ‘friends’ on the app
-
-● Pick 2 of your minimal requirements and break each of them down into ~2-5 smaller tasks!
-<br />&nbsp;&nbsp;    ○ Profile pages
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Form for adding & updating alias, name, etc.
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Change password
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Have Read/Watched list
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Show a user's Favourite section for books & Movies
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     A display of the total number of books/movies read/watched
-<br />
-<br />&nbsp;&nbsp;    ○ Home page
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Login/Registration button
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Trending books or movies
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Search bar for books
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Button to see your selected books
-<br />&nbsp;&nbsp;&nbsp;&nbsp;     ■     Profile pic
+# NextStory
+## Project Description: 
+Write 3-5 sentences on your project topic. (try to address all points below)
+- Who is it for?
+  - people who want to find their next movie or book
 
+- What will it do?
+  - search for movie or books to watch/read
+  - get recommendations
+  - add stories for future reading
+
+- What type of data will it store?
+  - user profile
+  - book / movie information
+  - reviews/ratings for books/movies
+
+- What will users be able to do with this data?
+  - search for movie or books to watch/read
+  - get recommendations
+  - get information about books/movies
+  - update their own profiles
+  - write reviews for books/movies
+
+- What is some additional functionality you can add/remove based on time constraints?
+  - Users should be able to see information of 'friends' on the app
+
+
+## Project task requirements:
+- 3-5 minimal requirements (will definitely complete)
+  1. Users should be able to register for, log in to & log out of a regular user/admin account and have/be able to update a profileUsers should be able to search & filter books & movies of interest
```

Comment:
this should be a separate line, right? 😅

Reply:
ah good catch I will fix this